bin/zuul

```diff
@@ -333,4 +312,48 @@ scout_browser(function(err, all_browsers) {
     });
 });
 
+function readLocalConfig(config) {
+    var yaml = path.join(process.cwd(), '.zuul.yml')
+    var js = path.join(process.cwd(), 'zuul.config.js')
+    var yamlExists = fs.existsSync(yaml)
+    var jsExists = fs.existsSync(js)
+    if (yamlExists && jsExists) {
+        console.error('both .zuul.yaml and zuul.config.js are found in the project directory, please choose one')
+        process.exit(1)
+    } else if (yamlExists) {
```

Comment:
Hm... I actually see it's not enforced even within this file. Will fix my code but won't do it for other places not to pollut PR with unrelated changes.